server/routers/bikeTrailsAPI.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const bikeController = require('../controllers/bikeTrailsController')
+const locationWeatherController = require('../controllers/locationWeatherController.js')
+
+const bikeTrailsRoute = express.Router();
+
+bikeTrailsRoute.get('/:zip', 
+locationWeatherController.getGeoCodeURL, locationWeatherController.getLocationInfo, bikeController.getTrails, 
```

Comment:
Could be just me, but the formatting in this file looks a bit strange. Does it look the same if you auto format -> option + shift + f?